cypress/cypress/integration/video.spec.ts

```diff
@@ -0,0 +1,145 @@
+
+The full terms of this copyright and license should always be found in the root directory of this software deliverable as "license.txt" and if these terms are not found with this software, please contact the USC Stevens Center for the full license.
+*/
+import { mockDefaultSetup, cyMockGQL } from "../support/helpers";
+const clint = require("../fixtures/clint.json");
+const carlos = require("../fixtures/carlos.json");
+const FAKE_STYLE_HEADER_LOGO =
+  "http://scribe.usc.edu/wp-content/uploads/2021/02/PrimShield_Word_SmUse_Gold-Wh_RGB-1.png";
+
+describe("Video Mentor", () => {
+  it("Display mentor-name card over the left-corner of the video", () => {
+    mockDefaultSetup(cy, {
+      config: { mentorsDefault: ["clint"] },
+      mentorData: [clint],
+      apiResponse: "response_with_feedback.json",
+    });
+    cy.visit("/");
+
+    cy.get("[data-cy=header]").should("have.attr", "data-mentor", "clint");
+    cy.get("[data-cy=header]").contains(
+      "Clinton Anderson: Nuclear Electrician's Mate"
+    );
+
+    cy.get("[data-cy=video-container").within(($hc) => {
+      cy.get("[data-cy=mentor-name-card]").should("exist");
+    });
+
+    cy.get("[data-cy=topic-2] button").trigger("mouseover").click();
+    cy.get("[data-cy=history-chat]").should("exist");
+    // write msgs
+    cy.get("[data-cy=input-field]").type("Question 1");
+    cy.get("[data-cy=input-send]").trigger("mouseover").click();
+    // display card over a corner of the video
+    cy.get("[data-cy=mentor-name-card]").should("exist");
+  });
+
+  it("Display mentor-name card & star over the left-corner of the video", () => {
+    mockDefaultSetup(cy, {
+      config: { mentorsDefault: ["clint", "carlos"] },
+      mentorData: [clint, carlos],
+      apiResponse: "response_with_feedback.json",
+      gqlQueries: [
+        cyMockGQL("UserQuestionSetFeedback", { userQuestionSetFeedback: null }),
+      ],
+    });
+    cy.intercept("**/questions/?mentor=clint&query=*", {
+      fixture: "response_with_feedback.json",
+    });
+    cy.visit("/");
+
+    // Mentor Card in the beggining
+    cy.get("[data-cy=video-container").within(($hc) => {
+      cy.get("[data-cy=mentor-name-card]").should("exist");
+      cy.get("[data-cy=mentorname-faveicon-wrapper]").within(($hc) => {
+        cy.get("[data-cy=mentor-name]")
+          .should("exist")
+          .contains("Clinton Anderson");
+        cy.get("[data-cy=fave-button]").should("exist");
+      });
+    });
+
+    cy.get("[data-cy=header]").should("have.attr", "data-mentor", "clint");
```

Comment:
I'm pretty sure you don't have to redo the `get` here just do this:

```typescript
cy.get("[data-cy=video-container").within(($hc) => {
      // ...
}).should("have.attr", "data-mentor", "clint");
```

Reply:
I think I do because that `within` is just for `data-cy=video-container` and `header` is not inside of that div